plugins/bufbuild/validate-cpp/v0.9.0/buf.plugin.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/bufbuild/validate-cpp
+plugin_version: v0.9.0
+source_url: https://github.com/bufbuild/protoc-gen-validate
+description: Generates C++ code to validate Protobuf messages using protoc-gen-validate constraints.
+output_languages:
```

Comment:
Removed default_opts.

Reply:
Can we remove `default_opts` in the other plugins for consistency?

- bufbuild/connect-go
- grpc/go
- protocolbuffers/go

Reply:
I was going to pick that up with #193.